boards/stm32f3discovery/Makefile.dep

```diff
@@ -1,4 +1,10 @@
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
   USEMODULE += lsm303dlhc
+  USEMODULE += l3gxxxx
+endif
+
+ifneq (,$(filter l3gxxxx,$(USEMODULE)))
+  USEMODULE += i3g4250d
```

Comment:
@benpicco Here we have a problem with different revisions of the board: L3GD20 (Rev. C01, D01), I3G4250D (Rev. E02)

Reply:
Then better leave a comment here, so the user knows what to do. (Or better yet, also add a note about this to `board.txt`)